.github/workflows/docker.yaml

```diff
@@ -38,27 +38,24 @@ jobs:
             type=sha,prefix=,format=long
```

Comment:
I'm confused how/why we can't see the failures from https://github.com/matrix-org/rust-synapse-compress-state/pull/158 ?

Do we need to require some jobs succeed in order to notice this?

Reply:
Indeed, and I wish you could just click on "build" there instead of needing to go to the Checks tab and clicking on the workflow. But it's a strong signal at the very least.